package.json

```diff
@@ -7,14 +7,17 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "codecov:e2e": "codecov -f ./coverage/coverage-final.json",
-    "codecov:unit": "codecov -f ./coverage/jest/coverage-final.json",
-    "codecov": "codecov -f coverage/coverage-final.json && codecov -f coverage/jest/coverage-final.json",
+    "codecov:e2e": "codecov -f ./jest-coverage/coverage-final.json",
+    "codecov:unit": "codecov -f ./jest-coverage/coverage-final.json",
+    "codecov": "codecov -f jest-coverage/coverage-final.json && codecov -f cypress-coverage/coverage-final.json",
+    "copy:reports": "mkdir -p reports/ && cp cypress-coverage/coverage-final.json reports/from-cypress.json && cp jest-coverage/coverage-final.json reports/from-jest.json",
+    "combine:reports": "mkdir -p reports/ && npx nyc merge reports && mv coverage.json .nyc_output/out.json",
```

Comment:
@vickywane I see your script to merge cov tests in .nyc_output/out.json, but I don't see where this combined file is used. For example, a txt version of it can be uploaded as the final github action coverage artifact. Currently the CI script only uploads the jest cov report.